### ISSUE TITLE
Addon-manager version needs bump for Ingress addon

### ISSUE THREAD

**Issue by donaldguy** [kubernetes/minikube#909]
Per post-close discussion in #611, `minikube addon enable ingress` is insufficient

I am a little new to k8s in general, or I would go for the seemingly simple PR. I tried running 
`kubectl --namespace kube-system delete pod kube-addon-manager-minikube` followed by a 
`kubectl create --namespace kube-system -f addon-manager.yaml` where `addon-manager.yaml` was https://github.com/kubernetes/minikube/blob/daa01582e351e03ef281dc53986c0333dcbabc1e/deploy/addons/addon-manager.yaml with the 5.1s replaced with 6.1s; and this did seem to spawn the services initially, but then kube-addon-manager-minikube reappeared and started wreaking some havoc. A `minikube disable addon-manager` seemed to clean this up mostly

One other note is that addon-manager 6.1 seems to have already rolled forward to StatefulSets over PetSets, I'm getting a bunch of 
```
2016-12-12T21:53:21.495299184Z error: error pruning namespaced object apps/v1beta1, Kind=StatefulSet: the server could not find the requested resource
2016-12-12T21:53:21.497446416Z WRN: == Failed to execute /usr/local/bin/kubectl  apply --namespace=kube-system -f /etc/kubernetes/addons     --prune=true -l kubernetes.io/cluster-service=true --recursive >/dev/null at 2016-12-12T21:53:21+0000. 1 tries remaining. ==
``` 
in the pod logs

**Minikube version** (use `minikube version`):
v0.13.1, built from source, daa01582e351e03ef281dc53986c0333dcbabc1e

**Comment by r2d4** [comment]
Thanks for catching this.  This was my fault.  I was using an upgraded version of the addon manager for testing.  This should be fixed in the next version when we merge #908.  Unfortunately, for now its difficult to upgrade the addon-manager component in place.  I'll see if I can find a workaround in the meantime before our next release.

**Comment by r2d4** [comment]
Fixed by #917.  The fix will be in minikube v0.14
